### DISCUSSION TITLE
Hierarchical coarse graining

### DISCUSSION THREAD

**Discussion by abakan-zz** [prody/ProDy#1880] [Archived Issues]
Here is a link to an attempt to implement the approach in http://online.liebertpub.com/doi/pdf/10.1089/cmb.2007.R015:
https://github.com/prody/ProDy/blob/f307936e2890821cb48c5b587fcbc5abc28b2ba7/lib/prody/future.py

With some effort it may become ready for integrating into `dynamics` module.

**Comment by jamesmkrieger** [comment]
I wonder why this never got integrated. We should definitely try this.

**Comment by jamesmkrieger** [comment]
I'm going to give it a try
